app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -237,22 +237,43 @@ def group_skipped?(name)
            x_edit_view_tb history_main ems_container_dashboard ems_infra_dashboard).include?(name)
   end
 
+  def cb_send_checked_list
+    return true if %w(generic_objects).include?(@display)
+  end
+
+  def cb_enabled_for_nested
+    return true if %w(generic_objects).include?(@display)
+  end
+
+  def cb_enabled_value_for_nested
+    if @display == 'generic_objects'
+      return false if @lastaction == 'generic_objects'
+      return true if @lastaction == 'generic_object'
+    end
```

Comment:
This condition can be written in much nicer way.

Btw what is the expected result of the method if `@display != 'generic_objects'`?